b2.js

```diff
@@ -258,6 +258,11 @@
         var fieldValue2 = $field.attr('value2');
         var isValidParam = true;
         var inverseValue = $field.attr('data-inverse-value');
+        /**
```

Comment:
Please conform to the comment convention in this file, always use // for comments